spec/requests/image_uploads_spec.rb

```diff
@@ -76,7 +76,7 @@
       before do
         sign_in user
         allow(Rails).to receive(:cache).and_return(memory_store)
```

Comment:
Since this stub is out of date now if we want to use the memory store we need to stub the `RedisRailsCache`
```ruby
RedisRailsCache = ActiveSupport::Cache::MemoryStore.new
```